README.md

```diff
@@ -0,0 +1,36 @@
+# Blog app
+
+In this project we will creare a fully functional website that will show the list of posts and empower readers to interact with them by adding comments and liking posts.
+
+## Built With
+- Ruby on Rails
+
+## Technologies used
+- Github
+- Gitflow
+- Visual Studio Code
+- Rubocop
+
```

Comment:
- [ ] Let's include all the necessary instructions required to set up and run a rails app, e.g cloning the repo, bundle install, etc.